x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/overview_status.tsx

```diff
@@ -51,24 +51,30 @@ export function OverviewStatus() {
             pending: 0,
```

Comment:
I honestly think this `switch` deserves to be a separate function call, this `useEffect` callback is yuge.